components/molecules/TableHeader/table-header.tsx

```diff
@@ -5,6 +5,9 @@ import Select from "components/atoms/Select/custom-select";
 import Title from "components/atoms/Typography/title";
 import ComponentDateFilter from "../ComponentDateFilter/component-date-filter";
 import PaginationResult from "../PaginationResults/pagination-result";
+import { useRouter } from "next/router";
+import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
+import { useDebounce } from "rooks";
```

Comment:
nit: move third-party imports to the top of the file

```suggestion
import { useRouter } from "next/router";
import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
import { useDebounce } from "rooks";
```